Elevation level
---
bool success
float64 ActualPosition
